source/partials/valid_test_account_numbers.erb

```diff
@@ -0,0 +1,16 @@
+<aside class="notice">
+When testing payments in the sandbox environment please make sure you use valid sort codes and
+account numbers as these are validated to ensure they accept Faster Payments prior to being sent.
```

Comment:
```suggestion
Account numbers and sort codes are validated prior to being processed to ensure they are accepted by the Faster Payments scheme. When testing payment in sandbox please use a valid account number and sort code or alternatively use one from the list below. This will ensure the correct behaviour of payments during your testing.
```